### PR TITLE
Add french translation

### PR DESCRIPTION
**Pull request by Incremental92**
French translation with some english terms kept for clarity.

### CODE REVIEW

File: lang/fr-fr/lang.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        'description' => 'Authentification par Token Web JSON.',
+    ],
+    'permissions' => [
+        'settings' => 'Gestion de JWTAuth',
+    ],
+    'settings' => [
+        'menu_label' => 'JWTAuth',
+        'menu_description' => 'Configuration de JWTAuth',
+        'tabs' => [
+            'urls' => 'Options d''URL',
```
Comment (rluders) [inline lang/fr-fr/lang.php:15]: I don't think that using the double `single quotes` here will work. The PHP string is already using `single quotes`, which would be better if you need to use the single quote character to the language if you replace the quotes that limit the string with double-quotes.

**Review by rluders — CHANGES_REQUESTED** [review]
There are some places where you are using single quotes inside the single quotes string on PHP. I don't think that it would be a good idea. How about replacing the string single quotes to use double quotes?

File: lang/fr-fr/lang.php (hunk 1 above)
Comment (rluders) [reply lang/fr-fr/lang.php:15]: For example, something like this:

```'urls' => "Options d''URL", ```

**Comment by Incremental92** [comment]
OK I will change it.

**Comment by Incremental92** [comment]
I made the correction.

**Pull request merged**